share/normalize/utils.py

```diff
@@ -36,3 +36,7 @@ def format_address(self, address1='', address2='', city='', state_or_province=''
         return '{}, {}'.format(city, state_or_province)
 
     return address1
+
+
+def str_to_int(amount):
```

Comment:
This might be better off as a link?

`Int(my.probably.an.number.value)`

I don't really have a strong opinion one way or the other.